spec/models/page_spec.rb

```diff
@@ -256,5 +263,62 @@
       expect(Page.new.action_count).to eq 0
     end
   end
+
+  describe 'friendly_id' do
```

Comment:
@Tuuleh let's rip out our meagre solution for slug generation. I've added some failing specs for behaviour I think we should support, now that we're using the `friendly_id` gem.